H5P.BranchedVideo/branched-video.js

```diff
@@ -930,17 +930,38 @@ H5P.BranchedVideo = (function ($) {
       volumeSlider.max = 100;
       volumeSlider.value = 100;
       volumeSlider.style.display = 'none';
+      var lastSliderValue = 100;
       volumeButton.onmouseenter = function(){
         volumeSlider.style.display = 'block';
         var currentVid = getBranch(self.currentPlayingSlug).getVideoHTML();
         var volume = currentVid.volume * 100;
         volumeSlider.value = volume;
       };
       volumeDiv.onmouseleave = function(){volumeSlider.style.display = 'none';};
+      volumeButton.onmousedown = function() {
+        if (volumeSlider.value == 0) {
+          //UNMUTE
+          volumeSlider.value = lastSliderValue;
+          volumeButton.className = 'tapestry-volume-button';
```

Comment:
Is this line necessary? (since you are already changing the class on line 960)

Reply:
You're right, the lines are unnecessary. How do I remove/edit them on Github?

Reply:
You'll need to make the changes on your branch and then commit the changes. You can use `git commit --amend` to just amend your changes to the current commit.

Reply:
Okay, cool, I'll do that now. Thanks!